gasket-driver/pkg.yaml

```diff
@@ -0,0 +1,36 @@
+name: gasket-driver-pkg
+variant: scratch
+shell: /toolchain/bin/bash
+dependencies:
+  - stage: kernel-build
+steps:
+  - sources:
+      - url: https://github.com/google/gasket-driver/archive/{{ .gasket_driver_ref }}.tar.gz
+        destination: gasket-driver.tar.gz
+        sha256: "{{ .gasket_driver_sha256 }}"
+        sha512: "{{ .gasket_driver_sha512 }}"
+    env:
+      ARCH: {{ if eq .ARCH "aarch64"}}arm64{{ else if eq .ARCH "x86_64" }}x86_64{{ else }}unsupported{{ end }}
+    prepare:
+      - |
+        tar xf gasket-driver.tar.gz --strip-components=1
+    build:
+      - |
+        cd src
+        sed -i 's|/lib/modules/$(KVERSION)/build|/src|' ./Makefile
+        make all
+
+    install:
+      - |
+        mkdir -p /rootfs/lib/modules/$(cat /src/include/config/kernel.release)/
+        
+        # create these so we don't get warnings when depmod runs
+        touch /rootfs/lib/modules/$(cat /src/include/config/kernel.release)/modules.order /rootfs/lib/modules/$(cat /src/include/config/kernel.release)/modules.builtin
+
+        make -C /src M=$(pwd)/src modules_install INSTALL_MOD_PATH=/rootfs
+
+        # remove these as we don't actually need/want them
+        rm /rootfs/lib/modules/$(cat /src/include/config/kernel.release)/modules.order /rootfs/lib/modules/$(cat /src/include/config/kernel.release)/modules.builtin
```

Comment:
sure this is not needed? the kmod library usually finds drivers from these files I believe

Reply:
From what I saw they remained empty files. Though I really only looked at modules.builtin. Initially when I had them still in there and installed the extension it overwrote the modules.builtin with an empty file. The modules seem to load just fine regardless of whether the file is empty or populated.